src/components/UI/InformationTabs/ApplicabilityEvidence.tsx

```diff
@@ -37,6 +38,21 @@ export default function ApplicabilityEvidence(props: Props): JSX.Element {
 						</div>
 						<Divider />
 					</>
+				) : (
+					// If CVE is not applicable
```

Comment:
what if not entitled? or other status?

Reply:
Currently, there is no logic for other status in the Applicability. I added the status for Applicable and not Applicable in VScode.
When I add the status I'll know what to add in the front.